pyproject.toml

```
@@ -72,8 +72,8 @@ include = ["/multipart", "/tests"]
 
 [tool.ruff]
 line-length = 120
-select = ["E", "F", "I", "FA"]
-ignore = ["B904", "B028", "F841", "E741"]
+lint.select = ["E", "F", "I", "FA"]
+lint.ignore = ["B904", "B028", "F841", "E741"]
 
```

What's this?

---

ruff said it is deprecated and ask for new format so I added it.

---

```
ruff multipart tests
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `pyproject.toml`:
  - 'ignore' -> 'lint.ignore'
  - 'select' -> 'lint.select'
```

---

it is same thing, just new style

---

Can we do this, instead?
```suggestion

[tool.ruff.lint]
select = ["E", "F", "I", "FA"]
ignore = ["B904", "B028", "F841", "E741"]

```

---

```
ruff failed
  Cause: Failed to parse python-multipart/pyproject.toml
  Cause: TOML parse error at line 76, column 1
   |
76 | [tool.ruff.lint]
   | ^^^^^^^^^^^^^^^^
unknown field `lint`
```

---

@onuralpszr, I suggest to split your PR in two PRs. ruff configuration isn't part of documentation.

Also, seems like your ruff is outdated. I've checked your branch with ruff 0.2.1 (as described in `pyproject.toml`) and it works well with `[tool.ruff.lint]`. I'd prefer to use the later way, it's clearer.

That's a working diff:

```diff
diff --git a/pyproject.toml b/pyproject.toml
index 41550c3..7668f9c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -72,12 +72,14 @@ include = ["/multipart", "/tests"]

 [tool.ruff]
 line-length = 120
-lint.select = ["E", "F", "I", "FA"]
-lint.ignore = ["B904", "B028", "F841", "E741"]

 [tool.ruff.format]
 skip-magic-trailing-comma = true

+[tool.ruff.lint]
+select = ["E", "F", "I", "FA"]
+ignore = ["B904", "B028", "F841", "E741"]
+
 [tool.ruff.lint.isort]
 combine-as-imports = true
 split-on-trailing-comma = false
```

---

@igoose1  I tried exact same and gives me previously, I re-tried it works yes and I removed from PR I will open new one